web-admin/src/features/errors/error-utils.ts

```diff
@@ -139,6 +142,12 @@ export function createGlobalErrorCallback(queryClient: QueryClient) {
       return;
     }
 
+    // Handle case when usage metrics project is not available
+    // This is always the case on non-prod
+    if (isMetricsProjectQuery(query)) {
```

Comment:
It's not clear to me why this function would return false outside of production?

Reply:
I have removed that comment. It could be down in prod as well.

Reply:
Oh, this is a custom API, so if the Rill API resource has not successfully reconciled, the API path will be undefined or something?

Reply:
Ya that could be the case on prod. Right now, on non-prod it is not preset at all.